app/connectors/SecureMessagingConnector.scala

```diff
@@ -26,7 +26,7 @@ import uk.gov.hmrc.http.HttpReads.Implicits.readRaw
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class SecureMessageFrontendConnector @Inject()(httpClient: HttpClient, config: SecureMessagingConfig)(implicit ec: ExecutionContext)
+class SecureMessagingConnector @Inject()(httpClient: HttpClient, config: SecureMessagingConfig)(implicit ec: ExecutionContext)
```

Comment:
Why remove the 'Frontend' part of the name? This connector integrates with the 'secure-message-frontend' service, not the 'secure-message' (backend) service.

Reply:
I consider `Frontend` only as an implementation detail and `SecureMessagingConnector` more in line with the other "Secure Messaging" classes/topic (Controller and Config). Anyway, not a big deal. I have renamed it to the old name.